src/storage/v3/property_value.hpp

```diff
@@ -330,6 +330,36 @@ inline bool operator<(const PropertyValue &first, const PropertyValue &second) {
   }
 }
 
+inline bool operator>=(const PropertyValue &first, const PropertyValue &second) {
+  if (!PropertyValue::AreComparableTypes(first.type(), second.type())) return first.type() >= second.type();
+  switch (first.type()) {
```

Comment:
This is needed for comparisons in indices

Reply:
Maybe exploit the fact that `(a >= b) <=> !(a < b)` to reduce code duplication and complexity.